### DISCUSSION TITLE
docs/genuine-oj/introduction

### DISCUSSION THREAD

**Discussion by giscus** [yxzlwz/blog#5] [Announcements]
# docs/genuine-oj/introduction

Genuine-OJ 是一个由 @yang-zhongtian (https://github.com/yang-zhongtian) @yxzlwz (https://github.com/yxzlwz) 等开发的开源 OJ 项目。 仓库地址 (https://github.com/genuine-oj); API 接口文档 (https://gen...

https://blog.yixiangzhilv.com/docs/genuine-oj/introduction.html

**Comment by stackedu** [comment]
各位大佬，有没有已经搭好的，想先试一试。

**Reply by yxzlwz** [reply]
你好，很高兴你关注我们的项目。很抱歉，目前我所了解的已部署的平台均为正式项目，且由于知名度过小暂不考虑搭建官方预览平台。如果确有需要，请与我邮件联系，我可以为你开设临时账号（联系方式见文档或我的GitHub主页）。

**Comment by GaIIagherdev** [comment]
你好，想试试demo
